### ISSUE TITLE
hw05 ready for grading 

### ISSUE THREAD

**Issue by ilgan** [ilgan/STAT545-hw-ganelin-ilya#4]

**Comment by ilgan** [comment]
@vincenzocoia @gvdr @ksedivyhaley @JoeyBernhardt @mynamedaike @pgonzaleze @derekcho

**Comment by tom-hc-park** [comment]
Hello, I am Hyeongcheol Park. Thank you for your mail and inviting me on your GitHub!
To sum up what I have learned from your work and what I was especially impressed(and a little suggestion);

1. Bold sentence mentioned the original questions that made me easy to remember the question what what’s going on your assignment very well.

2. Your idea of ageGap = maxLifeExp-minLifeExp for reordering country factor was interesting .

3. It was  very good idea to arrange country and lifeExp to show arrange function does not effect on the figure of the graph. Actually I didn't know how to show this when I was doing my homework. I think your way is good to show that :)

4. It was as nice to try many different version such as pdf, png, rsd, even csv and txt! And it was kind of you to give all the links for readers, which I didn't. 
 (Unfortunately, I failed to open RDS :( )


If I should give you some suggestion for your homework even though I actually do not find any flaw, I would say that it might be better;

1. It would be better if you write some more comments on reordering process and the effect of reorder on the graph by ggplot.

2. My simple suggestion.
h_continents <- c("Oceania")
h_gap <- gapminder %>%
  filter(continent != h_continents)

Instead of this, I think you may just delete Oceania by
  filter(continent != “Oceania”)

To sum up, it was very nice work. your homework contained new coding a lot! I enjoyed exploring your GitHub. Thank you and I hope you enjoyed learning!

**Comment by ilgan** [comment]
@aiod01 regarding teh second note. this type of coding is, as it was mentioned in the code I believe, was made in for future expansion of teh list. It is a common practice to aim for future expansion/generalization in CS in general.
But thank you for your suggestions! Very helpful!

**Comment by sevenelevenkevin** [comment]
**Overall:**
-great work!
-required number of tasks completed
-code was clear and concise

**Overall Suggestions:**
-knitr kable didn't output a table correctly, I think you have to add format='markdown'
-no reflection/progress report included
-maybe cite some resources you used?

**Suggestions on Specific Sections:**
Factor management
-all the tasks completed here but maybe include a bit more detail/description on what the code is doing (there are lot's of chunks of code together, maybe add some comments in between)

File I/O
-for saving and reading back in as csv, maybe describe some of the differences/changes this process makes to the dataset, such as if reordering factors survives the process

Visualization design
-didn't really compare an old and new plot incorporating the material on visualization design and color. only plot made was earlier exploring the reordering of factors

Writing figures to file
-the saving and linking part worked well but maybe check the size of your plots to make sure it's easy to look at, eg [my_plot pdf file size 1x2](https://github.com/ilgan/STAT545-hw-ganelin-ilya/blob/master/HW5/media/my_plot_2.pdf) was pretty big

---
Peer Reviewed by Kevin Zhu

**Comment by ilgan** [comment]
@k3vzhu 
- Regarding the plot sizes - it was intentional, just to show different sizes and the options, I wouldn't post the real work in unreadable format. 
- will add more descriptions in the future.
Thanks for the feedback!

**Comment by derekcho** [comment]
Hi @ilgan, here are some comments about your hw05:

-	The first part of the assignment was well done. You were able to successfully drop Oceania and its associated countries from gapminder.
-	You should use headers or something else in your report so it is easy to tell when you are transitioning from one section to another
-	When you reorder the factors of continent, you should specify a metric to reorder by rather than using the default (median in this case)
-	You include a lot of redundant and unnecessary code in your report where the results are not used elsewhere. 
-	Your factor reordering plot was well done. However, you need to provide more interpretations of your plots
-	What are the differences between the three read/write functions. You show them but there isn’t an explanation.
-	For visualization design, it isn’t clear what features you include from the visualization design class. You could have added a new color palette or theme to your plots. You need to list the differences between a new plot and an old plot or list the features that you include

Your grade will be emailed to you at a later date.
